README.md

```diff
@@ -28,7 +28,7 @@ A bundle will be produced from the native packager settings of this project. A b
 packager distribution and includes some component configuration. To load the bundle:
 
 ```bash
-loadBundle <HIT THE TAB KEY AND THEN RETURN>
+conductr:loadBundle <HIT THE TAB KEY AND THEN RETURN>
```

Comment:
With sbt-web we chose the approach that common commands were not namespaced in order to reduce typing.

When I think about it, the `conduct` command for the Python CLI is really about namespacing on the command line, so an sbt convention of namespacing all commands to align with the CLI sub commands sounds reasonable.

Let's just fix up the doc then and go with the namespaced approach then. :-)